cpp/include/rdma_proxy_client.h

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright (C) 2016 iNuron NV
+
+This file is part of Open vStorage Open Source Edition (OSE), as available from
+
+
+    http://www.openvstorage.org and
+    http://www.openvstorage.com.
+
+This file is free software; you can redistribute it and/or modify it
+under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+as published by the Free Software Foundation, in version 3 as it comes
+in the <LICENSE.txt> file of the Open vStorage OSE distribution.
+
+Open vStorage is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY of any kind.
+
+*/
+
+#pragma once
+#include "generic_proxy_client.h"
+
+namespace alba {
+namespace proxy_client {
+
+class RDMAProxy_client : public GenericProxy_client {
```

Comment:
2 reasons:
- it's shorter.
- there's no need for extra flexibility in this case.